python/fapolicy_analyzer/ui/trust_file_list.py

```diff
@@ -106,8 +107,39 @@ def on_addBtn_clicked(self, *args):
         )
         fcd.set_select_multiple(True)
         response = fcd.run()
+        fcd.hide()
```

Comment:
Should this be `destroy()` instead of `hide()`?

Reply:
It gets destroy()ed in its original location. The file selection dialog occasionally would remain on-top of the new whitespace info dlg so I wanted to hide it. We still need the fcd instance farther on down the page to get the file list.